aztk/models/models.py

```diff
@@ -109,11 +111,23 @@ def merge(self, other):
     def mixed_mode(self) -> bool:
         return self.vm_count > 0 and self.vm_low_pri_count > 0
 
+
+    def gpu_enabled(self):
+        return helpers.is_gpu_enabled(self.vm_size)
+
+    def get_docker_repo(self):
+        return self.toolkit.get_docker_repo(self.gpu_enabled())
+
     def validate(self) -> bool:
         """
         Validate the config at its current state.
         Raises: Error if invalid
         """
+        if self.toolkit is None:
+            raise error.InvalidModelError(
+                "Please supply a toolkit for the cluster")
```

Comment:
Maybe we should list available toolkits?

Reply:
It does if you input an invalid software

![image](https://user-images.githubusercontent.com/1031227/39215522-019a14b0-47cd-11e8-9cf8-71e6cedbb4df.png)